### PR TITLE
Add support for building Jax notebooks

### LINKED ISSUE #358: Support Flax notebooks
Hi, it looks like the doc-builder only supports building Colab notebooks in PyTorch/TensorFlow at the moment. Would it be possible to extend this for Flax as well? For context, we're trying to build some notebooks for Diffusers, which uses Flax in some of the training doc examples.

cc @yiyixuxu @sgugger 

https://github.com/huggingface/doc-builder/blob/10fd3bbeb0eea8cff685e67c7ee414a211bde8fc/src/doc_builder/convert_to_notebook.py#L89

### PR DESCRIPTION
**Pull request by stevhliu**
This PR fixes #358 and it'll allow the Diffusers docs to build Jax notebooks! 🎉

### CODE REVIEW

File: tests/test_convert_to_notebook.py [hunk 1]
```diff
@@ -148,6 +156,17 @@ def test_split_framewors(self):
 tf_sample
 ```
 End
+"""
+        jax_content = """
```
Comment (sgugger) [inline tests/test_convert_to_notebook.py:160]: Sshould be actually tested below ;-)

**Review by sgugger — COMMENTED** [review]
Thanks for working on this! I think the test is not testing anything though.

File: tests/test_convert_to_notebook.py (hunk 1 above)
Comment (stevhliu) [reply tests/test_convert_to_notebook.py:160]: D'oh! Should be tested now :)

**Review by sgugger — APPROVED** [review]
Thanks! Looks like it just needs a `make style` now.

**Pull request merged**